### PR TITLE
Hotfix/fix name column width

### PR DESCRIPTION
**Pull request by rogermiretin2**
### Changed
- In credential procedures list, add limit to name column width and change title to "Full name"
- In credential procedures list, change datetime format of "updated" column
- In credential procedures list, change pagination to 10/25/50 visible objects at a time.

### CODE REVIEW

File: src/assets/i18n/ca.json [hunk 1]
```diff
@@ -34,15 +34,15 @@
...
-    "name": "Name",
-    "updated": "Updated",
-    "sendReminder": "Send Reminder",
-    "cancel": "Cancel",
-    "create" : "Create Credential",
-    "buttonAdmin":"New credential as Signer"
+    "title": "Llista de credencials emeses",
+    "button": "Nova credencial",
+    "status": "Estatus",
+    "full_name": "Nom complet",
```
Comment (oriolcanades) [inline src/assets/i18n/ca.json:40]: Utilitzem fullName, has utilitzat snake_case quan la resta usa camelCase

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "title": "List of issued credentials",
     "button": "New credential",
     "status": "Status",
-    "name": "Name",
+    "full_name": "Full name",
```
Comment (oriolcanades) [inline src/assets/i18n/en.json:39]: ibid

File: src/assets/i18n/es.json [hunk 3]
```diff
@@ -34,15 +34,15 @@
...
-    "name": "Name",
-    "updated": "Updated",
-    "sendReminder": "Send Reminder",
-    "cancel": "Cancel",
-    "create" : "Create Credential",
-    "buttonAdmin":"New credential as Signer"
+    "title": "Lista de credenciales emitidas",
+    "button": "Nueva credencial",
+    "status": "Estatus",
+    "full_name": "Nombre completo",
```
Comment (oriolcanades) [inline src/assets/i18n/es.json:40]: ibid

**Review by oriolcanades — CHANGES_REQUESTED** [review]

File: src/assets/i18n/ca.json (hunk 1 above)
Comment (rogermiretin2) [reply src/assets/i18n/ca.json]: ok, fet

File: src/assets/i18n/en.json (hunk 2 above)
Comment (rogermiretin2) [reply src/assets/i18n/en.json]: fet

File: src/assets/i18n/es.json (hunk 3 above)
Comment (rogermiretin2) [reply src/assets/i18n/es.json]: fet

**Pull request merged**